code-challenges.test.js

```diff
@@ -23,45 +23,132 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // b) Create the function that makes the test pass.
 
+
+
+
+// describe("", () => {
+//     it("", () => {
+//         expect().toEqual()
+//     })
+// })
+
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+
+describe("divisibleBy3", () => {
+    it("checks to see if a number is divisible by 3", () => {
+        const object1 = { number: 15 }
+        // Expected output: "15 is divisible by three"
+        const object2 = { number: 0 }
+        // Expected output: "0 is divisible by three"
+        const object3 = { number: -7 }
+        // Expected output: "-7 is not divisible by three"
+        expect(divisibleBy3(object1)).toEqual("15 is divisible by three")
+        expect(divisibleBy3(object2)).toEqual("0 is divisible by three")
+        expect(divisibleBy3(object3)).toEqual("-7 is not divisible by three")
+    })
+})
 
 // b) Create the function that makes the test pass.
+const divisibleBy3 = (object) => {
+    let myNot = "not "
+    if (object.number % 3 === 0){
+        myNot = ""
+    }
+    return `${object.number} is ${myNot}divisible by three`
```

Comment:
Interesting approach, although it makes it a little more confusing to read right away